face_recognition_LDA.m

```diff
@@ -0,0 +1,32 @@
+
+% get directories for all images
+% please use 'getfn.m' with this script
+fn = getfn('C:\Users\json13\Desktop\att_faces', 'pgm$')
+
+% read and convert images to column vector
+image_dims = [112, 92];
+num_images = numel(fn);
+images = [];
+for n = 1:num_images
+    img = imread(fn{n});
+    if n == 1
+        images = zeros(prod(image_dims), num_images);
+    end
+    images(:, n) = img(:);
+end
+
+% split into training and testing data with 2:8 split ratio
+train_data = [];
+test_data = [];
+cont = 1;
+for i=1:42
```

Comment:
Can you use a variable here instead of hard coding 42, it'll lead to less mistakes later